hw/ip/i2c/data/i2c.hjson

```diff
@@ -57,8 +57,9 @@
     { name: "tx_overflow"
       desc: "target mode interrupt: raised if TX FIFO has overflowed."
     }
-    { name: "acq_overflow"
-      desc: "target mode interrupt: raised if ACQ FIFO has overflowed."
+    { name: "acq_full"
+      desc: "target mode interrupt: raised if ACQ FIFO becomes full.  This is a level status interrupt."
```

Comment:
I know I've heard "level status interrupt" but I forget what it means and can't find it documented. Does it mean the flag gets lowered automatically when it's no longer full? If that's the case and we still can overflow, then we'd want to keep an overflow interrupt to show that data's been lost.